src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -83,31 +83,85 @@ class PluginTableContainer:
     columns = attr.ib(default=attr.Factory(dict))
 
 
-@attr.s(frozen=True, slots=True)
-class TrendOutputDescription:
+@attr.s(kw_only=True)
+class _BaseTrendOutputDescription:
+    curve_names: List[str] = attr.ib(validator=list_of_strings)
+
+
+@attr.s()
+class PositionalPipeTrendDescription(_BaseTrendOutputDescription):
     """
-    .. include:: /alfacase_definitions/TrendOutputDescription.txt
+    .. include:: /alfacase_definitions/PositionalPipeTrendDescription.txt
 
     .. include:: /alfacase_definitions/list_of_unit_for_length.txt
     """
 
-    curve_names: List[str] = attr.ib(validator=list_of_strings)
     location = attrib_enum(type_=constants.OutputAttachmentLocation)
-    position = attrib_scalar(default=None, is_optional=True, category="length")
-    element_name: Optional[str] = attr.ib(
-        default=None, validator=optional(instance_of(str))
-    )
+    position = attrib_scalar(category="length")
+    element_name: str = attr.ib(default=None, validator=instance_of(str))
```

Comment:
This default value has to be removed if this is no longer optional.
```suggestion
    element_name: str = attr.ib(validator=instance_of(str))
```